build/sdk/js/templates/package.mustache

```diff
@@ -20,5 +20,8 @@
     "mocha": "^6.1.4",
     "sinon": "1.17.3",
     "uuid": "^3.3.2"
+  },
+  "resolutions": {
+    "*/**/js-yaml": "^3.13.1"
```

Comment:
what is this for?

Reply:
do you mean what is "resolutions" for or what is "js-yaml" for?

Reply:
`resolutions`

Reply:
discussed offline -- merging this